Automattic/wp-calypso
# Tiled gallery: add mosaic layout

Port of [PHP tiled galleries](https://github.com/Automattic/jetpack/tree/0d837791212dcfab0e75145a21857cc507e4c9d3/modules/tiled-gallery) algorithm to JS. 💪  

Implements "mosaic" aka "rectangular" layout in Tiled gallery block.

Code oddities inherited from PHP→JS porting will be cleaned up in follow-up PRs.

## Testing
- Enable beta blocks
- Add the block a post gutenpack-jn
- Add photos
- The rectangular layout is the default so you should see it right away:

![image](https://user-images.githubusercontent.com/87168/50137209-29757880-02a3-11e9-8323-b1d6ca953375.png)
